agora-test/Spec.hs

```diff
@@ -43,4 +53,7 @@ main =
       , testGroup
           "AuthorityToken tests"
           AuthorityToken.tests
+      , testGroup
```

Comment:
IIRC "Multisig tests" is currently broken. Multisig testgroup should be in the end for other tests to run.